lib/logstash/inputs/jdbc.rb

```diff
@@ -137,6 +137,9 @@ class LogStash::Inputs::Jdbc < LogStash::Inputs::Base
   # Whether to save state or not in last_run_metadata_path
   config :record_last_run, :validate => :boolean, :default => true
 
+  # Whether to force the lowercasing of identifier fields
+  config :force_lowercase_keys, :validate => :boolean, :default => true
```

Comment:
@jsvd to provide the same functionality we should set this to true, i know i added the comment but to keep consistent to the old behaviour, the default was to lowercase the keys. We should add this to the documentation, first thing that people will look into when running into this problem is if they are missing a setting.

BTW: should we call this setting force_lowecase_column_names?